test/integration/targets/vmware_guest_network/tasks/main.yml

```diff
@@ -91,7 +146,7 @@
   - name: assert the network adapter was disconnected
     assert:
       that:
-        - "disc_netadapter.changed == true"
```

Comment:
Here, the NIC are never connected. And so, this test fails. Any idea what's going on?

```
-*- mode: compilation; default-directory: "~/git_repos/ansible_projects/a/" -*-
Compilation started at Mon Sep 30 13:49:33

env VMWARE_TEST_PLATFORM=static ansible-test integration --python 3.7 -vvvv vmware_guest_network
RLIMIT_NOFILE: (1024, 524288)
MAXFD: -1
Using existing vcenter cloud config: test/integration/cloud-config-vcenter.ini
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/ ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.04728126525878906 seconds.
WARNING: PyYAML will be slow due to installation without libyaml support for interpreter: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/ ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.0221560001373291 seconds.
Run command: /usr/bin/python2 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python2
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.01591038703918457 seconds.
Run command: /usr/bin/python2.7 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python2.7
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.010446310043334961 seconds.
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.026368379592895508 seconds.
Run command: /usr/bin/python3.5 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python3.5
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.01976156234741211 seconds.
Run command: /usr/bin/python3.6 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python3.6
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.01885223388671875 seconds.
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3.7 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_da ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3.7
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.026102781295776367 seconds.
Run command: /usr/bin/python3.8 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python3.8
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.014789342880249023 seconds.
WARNING: A Python 3.5 interpreter was found at "/usr/bin/python3.5", yet a matching pip was not found.
WARNING: A Python 3.6 interpreter was found at "/usr/bin/python3.6", yet a matching pip was not found.
WARNING: A Python 3.8 interpreter was found at "/usr/bin/python3.8", yet a matching pip was not found.
>>> Environment Description
{
    "known_hosts_hash": "6ca96a42156670d6cd84225661960118",
    "pip_interpreters": {
        "": "#!/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3",
        "2": "#!/usr/bin/python2",
        "2.7": "#!/usr/bin/python2",
        "3": "#!/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3",
        "3.7": "#!/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3"
    },
    "pip_paths": {
        "": "/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/pip",
        "2": "/usr/bin/pip2",
        "2.6": null,
        "2.7": "/usr/bin/pip2.7",
        "3": "/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/pip3",
        "3.5": null,
        "3.6": null,
        "3.7": "/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/pip3.7",
        "3.8": null
    },
    "program_versions": {
        "": [
            "3.7.4 (default, Jul  9 2019, 16:32:37)",
            "[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]",
            "pip 19.2.3 from /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/lib/python3.7/site-packages/pip"
        ],
        "2": [
            "2.7.16 (default, Apr 30 2019, 15:54:43)",
            "[GCC 9.0.1 20190312 (Red Hat 9.0.1-0.10)]",
            "pip 19.0.3 from /usr/lib/python2.7/site-packages/pip"
        ],
        "2.7": [
            "2.7.16 (default, Apr 30 2019, 15:54:43)",
            "[GCC 9.0.1 20190312 (Red Hat 9.0.1-0.10)]",
            "pip 19.0.3 from /usr/lib/python2.7/site-packages/pip"
        ],
        "3": [
            "3.7.4 (default, Jul  9 2019, 16:32:37)",
            "[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]",
            "pip 19.2.3 from /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/lib/python3.7/site-packages/pip"
        ],
        "3.5": [
            "3.5.7 (default, Mar 19 2019, 18:04:30)",
            "[GCC 9.0.1 20190227 (Red Hat 9.0.1-0.8)]"
        ],
        "3.6": [
            "3.6.9 (default, Jul  3 2019, 15:08:07)",
            "[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]"
        ],
        "3.7": [
            "3.7.4 (default, Jul  9 2019, 16:32:37)",
            "[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]",
            "pip 19.2.3 from /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/lib/python3.7/site-packages/pip"
        ],
        "3.8": [
            "3.8.0b4 (default, Sep  2 2019, 08:06:41)",
            "[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]"
        ]
    },
    "python_paths": {
        "": "/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python",
        "2": "/usr/bin/python2",
        "2.6": null,
        "2.7": "/usr/bin/python2.7",
        "3": "/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3",
        "3.5": "/usr/bin/python3.5",
        "3.6": "/usr/bin/python3.6",
        "3.7": "/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3.7",
        "3.8": "/usr/bin/python3.8"
    },
    "warnings": [
        "A Python 3.5 interpreter was found at \"/usr/bin/python3.5\", yet a matching pip was not found.",
        "A Python 3.6 interpreter was found at \"/usr/bin/python3.6\", yet a matching pip was not found.",
        "A Python 3.8 interpreter was found at \"/usr/bin/python3.8\", yet a matching pip was not found."
    ]
}
Running vmware_guest_network integration test role
Preparing temporary directory: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ
binary_modules:
  binary_modules_posix
  binary_modules_winrm
cloudscale_common:
  cloudscale_floating_ip
  cloudscale_server
  cloudscale_server_group
  cloudscale_volume
  inventory_cloudscale
connection:
  connection_buildah
  connection_chroot
  connection_docker
  connection_jail
  connection_libvirt_lxc
  connection_local
  connection_lxc
  connection_lxd
  connection_paramiko_ssh
  connection_podman
  connection_posix
  connection_psrp
  connection_ssh
  connection_windows_ssh
  connection_winrm
connection_posix:
  connection_buildah
  connection_chroot
  connection_docker
  connection_jail
  connection_libvirt_lxc
  connection_local
  connection_lxc
  connection_lxd
  connection_paramiko_ssh
  connection_podman
  connection_ssh
cs_common:
  cs_account
  cs_affinitygroup
  cs_cluster
  cs_configuration
  cs_disk_offering
  cs_domain
  cs_firewall
  cs_host
  cs_image_store
  cs_instance
  cs_instance_info
  cs_instance_nic
  cs_instance_nic_secondaryip
  cs_instance_password_reset
  cs_instancegroup
  cs_ip_address
  cs_iso
  cs_loadbalancer_rule
  cs_network
  cs_network_acl
  cs_network_acl_rule
  cs_network_offering
  cs_physical_network
  cs_pod
  cs_portforward
  cs_project
  cs_region
  cs_resourcelimit
  cs_role
  cs_role_permission
  cs_router
  cs_securitygroup
  cs_securitygroup_rule
  cs_service_offering
  cs_snapshot_policy
  cs_sshkeypair
  cs_storage_pool
  cs_template
  cs_traffic_type
  cs_user
  cs_vlan_ip_range
  cs_vmsnapshot
  cs_volume
  cs_vpc
  cs_vpc_offering
  cs_vpn_connection
  cs_vpn_customer_gateway
  cs_vpn_gateway
  cs_zone
  cs_zone_info
include_parent_role_vars:
  special_vars
prepare_eos_tests:
  eos_banner
  eos_bgp
  eos_command
  eos_config
  eos_eapi
  eos_facts
  eos_interface
  eos_interfaces
  eos_l2_interface
  eos_l2_interfaces
  eos_l3_interface
  eos_l3_interfaces
  eos_lacp
  eos_lacp_interfaces
  eos_lag_interfaces
  eos_linkagg
  eos_lldp
  eos_lldp_global
  eos_lldp_interfaces
  eos_logging
  eos_smoke
  eos_static_route
  eos_system
  eos_user
  eos_vlan
  eos_vlans
  eos_vrf
prepare_http_tests:
  get_certificate
  get_url
  lookups
  uri
  win_get_url
  win_uri
prepare_ios_tests:
  ios_banner
  ios_bgp
  ios_command
  ios_config
  ios_facts
  ios_interface
  ios_l2_interface
  ios_l3_interface
  ios_linkagg
  ios_lldp
  ios_logging
  ios_ntp
  ios_ping
  ios_smoke
  ios_static_route
  ios_system
  ios_user
  ios_vlan
prepare_iosxr_tests:
  iosxr_banner
  iosxr_bgp
  iosxr_command
  iosxr_config
  iosxr_facts
  iosxr_interface
  iosxr_logging
  iosxr_netconf
  iosxr_smoke
  iosxr_system
  iosxr_user
  netconf_config
  netconf_get
  netconf_rpc
prepare_junos_tests:
  junos_banner
  junos_command
  junos_config
  junos_facts
  junos_interface
  junos_interfaces
  junos_l2_interface
  junos_l2_interfaces
  junos_l3_interface
  junos_l3_interfaces
  junos_lacp
  junos_lag_interfaces
  junos_linkagg
  junos_lldp
  junos_lldp_global
  junos_lldp_interface
  junos_lldp_interfaces
  junos_logging
  junos_rpc
  junos_smoke
  junos_static_route
  junos_system
  junos_user
  junos_vlan
  junos_vrf
  netconf_config
  netconf_get
  netconf_rpc
prepare_nios_tests:
  nios_a_record
  nios_aaaa_record
  nios_cname_record
  nios_dns_view
  nios_host_record
  nios_mx_record
  nios_naptr_record
  nios_network
  nios_network_view
  nios_ptr_record
  nios_srv_record
  nios_txt_record
  nios_zone
prepare_nuage_tests:
  nuage_vspk
prepare_nxos_tests:
  nxos_aaa_server
  nxos_aaa_server_host
  nxos_acl
  nxos_acl_interface
  nxos_banner
  nxos_become
  nxos_bfd_global
  nxos_bfd_interfaces
  nxos_bgp
  nxos_bgp_af
  nxos_bgp_neighbor
  nxos_bgp_neighbor_af
  nxos_command
  nxos_config
  nxos_evpn_global
  nxos_evpn_vni
  nxos_facts
  nxos_feature
  nxos_gir
  nxos_gir_profile_management
  nxos_hsrp
  nxos_igmp
  nxos_igmp_interface
  nxos_igmp_snooping
  nxos_interface
  nxos_interface_ospf
  nxos_interfaces
  nxos_l2_interface
  nxos_l2_interfaces
  nxos_l3_interface
  nxos_l3_interfaces
  nxos_lacp
  nxos_lacp_interfaces
  nxos_lag_interfaces
  nxos_linkagg
  nxos_lldp
  nxos_lldp_global
  nxos_logging
  nxos_ntp
  nxos_ntp_auth
  nxos_ntp_options
  nxos_nxapi
  nxos_nxapi_transports
  nxos_ospf
  nxos_ospf_vrf
  nxos_overlay_global
  nxos_pim
  nxos_pim_interface
  nxos_pim_rp_address
  nxos_reboot
  nxos_rollback
  nxos_rpm
  nxos_smoke
  nxos_snapshot
  nxos_snmp_community
  nxos_snmp_contact
  nxos_snmp_host
  nxos_snmp_location
  nxos_snmp_traps
  nxos_snmp_user
  nxos_static_route
  nxos_system
  nxos_telemetry
  nxos_udld
  nxos_udld_interface
  nxos_user
  nxos_vlan
  nxos_vpc
  nxos_vpc_interface
  nxos_vrf
  nxos_vrf_af
  nxos_vrf_interface
  nxos_vrrp
  nxos_vtp_domain
  nxos_vtp_password
  nxos_vtp_version
  nxos_vxlan_vtep
  nxos_vxlan_vtep_vni
prepare_ovs_tests:
  openvswitch_bridge
  openvswitch_db
prepare_sros_tests:
  netconf_get
prepare_tests:
  apache2_module
  apt
  apt_key
  apt_repository
  archive
  assemble
  async
  async_fail
  at
  authorized_key
  aws_api_gateway
  aws_elasticbeanstalk_app
  aws_kms
  aws_lambda
  binary
  blockinfile
  changed_when
  command_shell
  copy
  debconf
  deploy_helper
  dnf
  ec2_ami
  ec2_eip
  ec2_elb_lb
  ec2_group
  ec2_instance
  ec2_key
  ec2_launch_template
  ec2_metadata_facts
  ec2_tag
  ec2_vol
  ec2_vol_info
  ec2_vpc_net
  ec2_vpc_route_table
  ec2_vpc_subnet
  ecs_certificate
  ecs_domain
  ecs_ecr
  elb_classic_lb
  elb_network_lb
  facts_d
  fetch
  file
  filters
  find
  flatpak
  flatpak_remote
  gem
  get_url
  getent
  git
  group
  hg
  iam_group
  iam_saml_federation
  ignore_errors
  ignore_unreachable
  iso_extract
  iterators
  known_hosts
  lambda_policy
  lineinfile
  locale_gen
  lookups
  package
  patch
  pip
  raw
  rds_param_group
  redshift
  replace
  s3_bucket_notification
  script
  service
  special_vars
  stat
  sts_assume_role
  subversion
  synchronize
  sysctl
  systemd
  template
  template_jinja2_latest
  unarchive
  uri
  user
  wait_for
  yum
  zypper
  zypper_repository
prepare_vmware_tests:
  vcenter_folder
  vcenter_license
  vmware_about_facts
  vmware_about_info
  vmware_cluster
  vmware_cluster_drs
  vmware_cluster_facts
  vmware_cluster_ha
  vmware_cluster_info
  vmware_cluster_vsan
  vmware_content_deploy_template
  vmware_content_library_info
  vmware_content_library_manager
  vmware_datacenter
  vmware_datastore_cluster
  vmware_datastore_facts
  vmware_datastore_info
  vmware_datastore_maintenancemode
  vmware_drs_group
  vmware_drs_group_facts
  vmware_drs_group_info
  vmware_drs_rule_facts
  vmware_drs_rule_info
  vmware_dvs_portgroup
  vmware_dvs_portgroup_facts
  vmware_dvs_portgroup_find
  vmware_dvs_portgroup_info
  vmware_dvswitch
  vmware_dvswitch_nioc
  vmware_dvswitch_pvlans
  vmware_dvswitch_uplink_pg
  vmware_evc_mode
  vmware_folder_info
  vmware_guest
  vmware_guest_boot_facts
  vmware_guest_boot_info
  vmware_guest_custom_attribute_defs
  vmware_guest_custom_attributes
  vmware_guest_customization_facts
  vmware_guest_customization_info
  vmware_guest_disk
  vmware_guest_disk_facts
  vmware_guest_disk_info
  vmware_guest_facts
  vmware_guest_find
  vmware_guest_info
  vmware_guest_move
  vmware_guest_network
  vmware_guest_powerstate
  vmware_guest_screenshot
  vmware_guest_sendkey
  vmware_guest_snapshot
  vmware_guest_snapshot_facts
  vmware_guest_snapshot_info
  vmware_guest_tools_wait
  vmware_host
  vmware_host_acceptance
  vmware_host_active_directory
  vmware_host_capability_facts
  vmware_host_capability_info
  vmware_host_config_facts
  vmware_host_config_info
  vmware_host_config_manager
  vmware_host_datastore
  vmware_host_dns_facts
  vmware_host_dns_info
  vmware_host_facts
  vmware_host_feature_facts
  vmware_host_feature_info
  vmware_host_firewall_facts
  vmware_host_firewall_info
  vmware_host_firewall_manager
  vmware_host_hyperthreading
  vmware_host_ipv6
  vmware_host_kernel_manager
  vmware_host_ntp
  vmware_host_ntp_facts
  vmware_host_ntp_info
  vmware_host_package_facts
  vmware_host_package_info
  vmware_host_powermgmt_policy
  vmware_host_powerstate
  vmware_host_scanhba
  vmware_host_service_facts
  vmware_host_service_info
  vmware_host_service_manager
  vmware_host_snmp
  vmware_host_ssl_facts
  vmware_host_ssl_info
  vmware_host_vmhba_facts
  vmware_host_vmhba_info
  vmware_host_vmnic_facts
  vmware_host_vmnic_info
  vmware_inventory
  vmware_local_role_facts
  vmware_local_role_info
  vmware_local_role_manager
  vmware_local_user_facts
  vmware_local_user_info
  vmware_local_user_manager
  vmware_maintenancemode
  vmware_portgroup
  vmware_portgroup_facts
  vmware_portgroup_info
  vmware_resource_pool
  vmware_resource_pool_facts
  vmware_resource_pool_info
  vmware_target_canonical_facts
  vmware_target_canonical_info
  vmware_vcenter_settings
  vmware_vcenter_statistics
  vmware_vm_facts
  vmware_vm_host_drs_rule
  vmware_vm_info
  vmware_vm_storage_policy_info
  vmware_vm_vm_drs_rule
  vmware_vmkernel
  vmware_vmkernel_facts
  vmware_vmkernel_info
  vmware_vsan_health_info
  vmware_vspan_session
  vmware_vswitch
  vmware_vswitch_facts
  vmware_vswitch_info
prepare_vyos_tests:
  vyos_interfaces
  vyos_l3_interfaces
  vyos_lag_interfaces
  vyos_lldp_interfaces
prepare_win_tests:
  win_certificate_store
  win_credential
  win_file
  win_find
  win_lineinfile
  win_reg_stat
  win_region
  win_regmerge
  win_template
  win_xml
setup_acme:
  acme_account
  acme_account_info
  acme_certificate
  acme_certificate_revoke
  acme_challenge_cert_helper
  acme_inspect
setup_azure:
  azure_rm_acs
  azure_rm_aks
  azure_rm_appgateway
  azure_rm_appserviceplan
  azure_rm_automationaccount
  azure_rm_autoscale
  azure_rm_availabilityset
  azure_rm_azurefirewall
  azure_rm_batchaccount
  azure_rm_cdnprofile
  azure_rm_containerinstance
  azure_rm_containerregistry
  azure_rm_cosmosdbaccount
  azure_rm_deployment
  azure_rm_devtestlab
  azure_rm_dnszone
  azure_rm_functionapp
  azure_rm_gallery
  azure_rm_hdinsightcluster
  azure_rm_image
  azure_rm_iothub
  azure_rm_keyvault
  azure_rm_loadbalancer
  azure_rm_lock
  azure_rm_manageddisk
  azure_rm_mariadbserver
  azure_rm_monitorlogprofile
  azure_rm_mysqlserver
  azure_rm_networkinterface
  azure_rm_postgresqlserver
  azure_rm_publicipaddress
  azure_rm_rediscache
  azure_rm_resource
  azure_rm_resourcegroup
  azure_rm_roledefinition
  azure_rm_routetable
  azure_rm_securitygroup
  azure_rm_servicebus
  azure_rm_sqlserver
  azure_rm_storageaccount
  azure_rm_storageblob
  azure_rm_subnet
  azure_rm_trafficmanagerprofile
  azure_rm_virtualmachineextension
  azure_rm_virtualmachineimage_facts
  azure_rm_virtualmachinescaleset
  azure_rm_virtualnetwork
  azure_rm_virtualnetworkgateway
  azure_rm_virtualnetworkpeering
  azure_rm_webapp
  azure_rm_workspace
setup_deb_repo:
  apt
setup_docker:
  docker_config
  docker_container
  docker_container_info
  docker_host_info
  docker_image
  docker_image_info
  docker_network
  docker_network_info
  docker_node
  docker_node_info
  docker_prune
  docker_secret
  docker_stack
  docker_swarm
  docker_swarm_info
  docker_swarm_service
  docker_swarm_service_info
  docker_volume
  docker_volume_info
  inventory_docker_machine
  inventory_docker_swarm
setup_ec2:
  aws_api_gateway
  aws_elasticbeanstalk_app
  aws_kms
  aws_lambda
  ec2_ami
  ec2_eip
  ec2_elb_lb
  ec2_group
  ec2_instance
  ec2_key
  ec2_launch_template
  ec2_metadata_facts
  ec2_tag
  ec2_vol
  ec2_vol_info
  ec2_vpc_net
  ec2_vpc_route_table
  ec2_vpc_subnet
  ecs_ecr
  elb_classic_lb
  elb_network_lb
  iam_group
  iam_saml_federation
  lambda_policy
  rds_param_group
  redshift
  s3_bucket_notification
  sts_assume_role
setup_epel:
  dnf
  docker_config
  docker_container
  docker_container_info
  docker_host_info
  docker_image
  docker_image_info
  docker_network
  docker_network_info
  docker_node
  docker_node_info
  docker_prune
  docker_secret
  docker_stack
  docker_swarm
  docker_swarm_info
  docker_swarm_service
  docker_swarm_service_info
  docker_volume
  docker_volume_info
  inventory_docker_machine
  inventory_docker_swarm
  setup_docker
  setup_rpm_repo
  ufw
  yum
setup_flatpak_remote:
  flatpak_remote
setup_influxdb:
  influxdb_user
setup_mongodb:
  mongodb_replicaset
  mongodb_shard
setup_mosquitto:
  mqtt
setup_mysql_db:
  mysql_db
  mysql_info
  mysql_user
  mysql_variables
  setup_zabbix
  zabbix_host
setup_nobody:
  copy
  file
  template
  template_jinja2_latest
setup_opennebula:
  one_host
setup_openssl:
  consul
  docker_swarm
  ecs_certificate
  get_certificate
  lookup_hashi_vault
  openssl_certificate
  openssl_certificate_info
  openssl_csr
  openssl_csr_info
  openssl_pkcs12
  openssl_privatekey
  openssl_privatekey_info
  openssl_publickey
setup_paramiko:
  connection_paramiko_ssh
setup_passlib:
  subversion
  vars_prompt
setup_pexpect:
  pause
  vars_prompt
setup_podman:
  podman_image
  podman_image_info
setup_postgresql_db:
  postgresql
setup_rabbitmq:
  rabbitmq_binding
  rabbitmq_lookup
  rabbitmq_plugin
  rabbitmq_publish
  rabbitmq_user
  rabbitmq_vhost
  rabbitmq_vhost_limits
setup_remote_constraints:
  docker_config
  docker_container
  docker_container_info
  docker_host_info
  docker_image
  docker_image_info
  docker_network
  docker_network_info
  docker_node
  docker_node_info
  docker_prune
  docker_secret
  docker_stack
  docker_swarm
  docker_swarm_info
  docker_swarm_service
  docker_swarm_service_info
  docker_volume
  docker_volume_info
  inventory_docker_machine
  inventory_docker_swarm
  setup_docker
  uri
setup_remote_tmp_dir:
  apt
  aws_eks
  certificate_complete_chain
  connection_windows_ssh
  dnf
  docker_config
  docker_container
  docker_container_info
  docker_host_info
  docker_image
  docker_image_info
  docker_network
  docker_network_info
  docker_node
  docker_node_info
  docker_prune
  docker_secret
  docker_stack
  docker_swarm
  docker_swarm_info
  docker_swarm_service
  docker_swarm_service_info
  docker_volume
  docker_volume_info
  ec2_instance
  ecs_cluster
  elb_application_lb
  fetch
  filesystem
  flatpak_remote
  get_certificate
  get_url
  ini_file
  inventory_docker_machine
  inventory_docker_swarm
  jboss
  k8s
  lookups
  lvg
  mongodb_replicaset
  mongodb_shard
  mysql_info
  mysql_user
  old_style_modules_posix
  prepare_http_tests
  rabbitmq_publish
  setup_deb_repo
  setup_docker
  setup_flatpak_remote
  setup_remote_constraints
  setup_wildfly_server
  setup_win_psget
  task_ordering
  unarchive
  uri
  want_json_modules_posix
  win_dsc
  win_format
  win_get_url
  win_hosts
  win_psexec
  win_psmodule
  win_psrepository
  win_uri
  xfs_quota
  yum
setup_rpm_repo:
  dnf
  yum
setup_ssh_keygen:
  openssh_cert
  openssh_keypair
setup_sshkey:
  ec2_key
  hcloud_ssh_key
  hcloud_ssh_key_info
setup_tls:
  mqtt
  rabbitmq_binding
  rabbitmq_lookup
  rabbitmq_plugin
  rabbitmq_publish
  rabbitmq_user
  rabbitmq_vhost
  rabbitmq_vhost_limits
  setup_mosquitto
  setup_rabbitmq
setup_wildfly_server:
  jboss
setup_win_device:
  win_dns_client
  win_netbios
setup_win_psget:
  win_psmodule
  win_psrepository
setup_zabbix:
  zabbix_host
template:
  template_jinja2_latest
Copying test/integration/targets/prepare_vmware_tests/ to /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/
Copying test/integration/targets/vmware_guest_network/ to /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/
Copying /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/ansible.cfg to /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/integration.cfg
Copying /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/inventory to /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory
>>> Playbook: vmware_guest_network-25645cr0.yml
[
    {
        "environment": {
            "ESXI1_HOSTNAME": "esxi1.test",
            "ESXI1_PASSWORD": "!234AaAa56",
            "ESXI1_USERNAME": "root",
            "ESXI2_HOSTNAME": "test2.test",
            "ESXI2_PASSWORD": "!234AaAa56",
            "ESXI2_USERNAME": "root",
            "VCENTER_HOSTNAME": "vcenter.test",
            "VCENTER_PASSWORD": "!234AaAa56",
            "VCENTER_USERNAME": "administrator@vsphere.local",
            "VMWARE_VALIDATE_CERTS": "false"
        },
        "gather_facts": true,
        "hosts": "testhost",
        "module_defaults": null,
        "roles": [
            "vmware_guest_network"
        ],
        "vars": {
            "esxi1_hostname": "esxi1.test",
            "esxi1_password": "!234AaAa56",
            "esxi1_username": "root",
            "esxi2_hostname": "test2.test",
            "esxi2_password": "!234AaAa56",
            "esxi2_username": "root",
            "output_dir": "/home/goneri/git_repos/ansible_projects/a/test/results/.tmp/output_dir",
            "resource_prefix": "ansible-test-t580-gone-11685378",
            "vcenter_hostname": "vcenter.test",
            "vcenter_password": "!234AaAa56",
            "vcenter_username": "administrator@vsphere.local",
            "vmware_validate_certs": "false"
        },
        "vars_files": []
    }
]
Run command: ansible-playbook vmware_guest_network-25645cr0.yml -i inventory -vvvv
Working directory: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/ansible-playbook
ANSIBLE_CALLBACK_WHITELIST=junit
ANSIBLE_CONFIG=/home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/integration.cfg
ANSIBLE_DEPRECATION_WARNINGS=false
ANSIBLE_FORCE_COLOR=true
ANSIBLE_FORCE_HANDLERS=true
ANSIBLE_HOST_KEY_CHECKING=false
ANSIBLE_HOST_PATTERN_MISMATCH=error
ANSIBLE_INVENTORY=/dev/null
ANSIBLE_LIBRARY=/dev/null
ANSIBLE_PYTHON_INTERPRETER=/set/ansible_python_interpreter/in/inventory
ANSIBLE_PYTHON_MODULE_RLIMIT_NOFILE=1024
ANSIBLE_RETRY_FILES_ENABLED=false
ANSIBLE_ROLES_PATH=/home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets
ANSIBLE_TEST_CI=
ANSIBLE_TEST_COVERAGE=
ANSIBLE_TEST_PYTHON_INTERPRETER=/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python
ANSIBLE_TEST_PYTHON_VERSION=3.7
HOME=/home/goneri
INVENTORY_PATH=/home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory
JUNIT_OUTPUT_DIR=/home/goneri/git_repos/ansible_projects/a/test/results/junit
LC_ALL=en_US.UTF-8
OUTPUT_DIR=/home/goneri/git_repos/ansible_projects/a/test/results/.tmp/output_dir
PAGER=/bin/cat
PATH=/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/injector:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
PYTHONPATH=/home/goneri/git_repos/ansible_projects/a/lib
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
ansible-playbook 2.10.0.dev0
  config file = /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/integration.cfg
  configured module search path = ['/dev/null']
  ansible python module location = /home/goneri/git_repos/ansible_projects/a/lib/ansible
  executable location = /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/ansible-playbook
  python version = 3.7.4 (default, Jul  9 2019, 16:32:37) [GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]
Using /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/integration.cfg as config file
setting up inventory plugins
host_list declined parsing /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory as it did not pass its verify_file() method
script declined parsing /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory as it did not pass its verify_file() method
auto declined parsing /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory as it did not pass its verify_file() method
Parsed /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory inventory source with ini plugin
Loading callback plugin default of type stdout, v2.0 from /home/goneri/git_repos/ansible_projects/a/lib/ansible/plugins/callback/default.py
Loading callback plugin junit of type aggregate, v2.0 from /home/goneri/git_repos/ansible_projects/a/lib/ansible/plugins/callback/junit.py
 [WARNING]: The `junit_xml` python module is not installed. Disabling the `junit` callback plugin.


PLAYBOOK: vmware_guest_network-25645cr0.yml ******************************************************************************************************************
Positional arguments: vmware_guest_network-25645cr0.yml
verbosity: 4
connection: smart
timeout: 10
force_handlers: True
become_method: sudo
tags: ('all',)
inventory: ('/home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/inventory',)
forks: 5
1 plays in vmware_guest_network-25645cr0.yml

PLAY [testhost] **********************************************************************************************************************************************

TASK [Gathering Facts] ***************************************************************************************************************************************
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356 `" && echo ansible-tmp-1569865776.9202282-140621768248356="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/system/setup.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp3apcvnnp TO /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356/AnsiballZ_setup.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356/ /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356/AnsiballZ_setup.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356/AnsiballZ_setup.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865776.9202282-140621768248356/ > /dev/null 2>&1 && sleep 0'
ok: [testhost]
META: ran handlers

TASK [prepare_vmware_tests : load vmware common vars] ********************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/main.yml:3
ok: [testhost] => {
    "ansible_facts": {
        "ccr1": "DC0_C0",
        "dc1": "DC0",
        "ds1": "LocalDS_0",
        "ds2": "LocalDS_1",
        "dvswitch1": "DVS0",
        "esxi1": "{{ esxi_hosts[0] }}",
        "esxi2": "{{ esxi_hosts[1] }}",
        "esxi3": "{{ esxi_hosts[2] }}",
        "esxi_user": "root",
        "f0": "F0",
        "switch1": "switch1"
    },
    "ansible_included_var_files": [
        "/home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/vars/common.yml"
    ],
    "changed": false
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/main.yml:7
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml for testhost

TASK [prepare_vmware_tests : load vars] **********************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:2
ok: [testhost] => {
    "ansible_facts": {
        "cluster_category": "test_cluster_cat_0001",
        "cluster_tag": "test_cluster_tag_0001",
        "esxi_hosts": [
            "esxi1.test",
            "esxi2.test"
        ],
        "esxi_password": "{{ esxi1_password }}",
        "infra": {
            "datastores": {
                "LocalDS_0": {
                    "path": "/srv/share/isos",
                    "ro": false,
                    "server": "gateway.test",
                    "type": "nfs"
                },
                "LocalDS_1": {
                    "path": "/srv/share/vms",
                    "ro": false,
                    "server": "gateway.test",
                    "type": "nfs"
                }
            }
        },
        "virtual_machines": [
            {
                "folder": "{{ f0 }}",
                "name": "DC0_H0_VM0"
            },
            {
                "folder": "{{ f0 }}",
                "name": "DC0_H0_VM1"
            }
        ],
        "virtual_machines_in_cluster": [
            {
                "cluster": "{{ ccr1 }}",
                "folder": "{{ f0 }}",
                "name": "DC0_C0_RP0_VM0"
            },
            {
                "cluster": "{{ ccr1 }}",
                "folder": "{{ f0 }}",
                "name": "DC0_C0_RP0_VM1"
            }
        ]
    },
    "ansible_included_var_files": [
        "/home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/vars/real_lab.yml"
    ],
    "changed": false
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:6
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml for testhost

TASK [prepare_vmware_tests : Remove the VM prepared by prepare_vmware_tests] *********************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:2
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110 `" && echo ansible-tmp-1569865779.7381935-37148917190110="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpxiqpp1k6 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110/ /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865779.7381935-37148917190110/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_H0_VM0', 'folder': 'F0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_H0_VM0",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "folder": "F0",
        "name": "DC0_H0_VM0"
    }
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560 `" && echo ansible-tmp-1569865782.2263072-246421041361560="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp5rwuqja5 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560/ /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865782.2263072-246421041361560/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_H0_VM1', 'folder': 'F0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_H0_VM1",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "folder": "F0",
        "name": "DC0_H0_VM1"
    }
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363 `" && echo ansible-tmp-1569865784.7719746-277932370723363="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp_elx3mqv TO /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363/ /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865784.7719746-277932370723363/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_C0_RP0_VM0', 'folder': 'F0', 'cluster': 'DC0_C0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_C0_RP0_VM0",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "cluster": "DC0_C0",
        "folder": "F0",
        "name": "DC0_C0_RP0_VM0"
    }
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061 `" && echo ansible-tmp-1569865786.9809036-280973636473061="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpea5rse2r TO /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061/ /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865786.9809036-280973636473061/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_C0_RP0_VM1', 'folder': 'F0', 'cluster': 'DC0_C0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_C0_RP0_VM1",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "cluster": "DC0_C0",
        "folder": "F0",
        "name": "DC0_C0_RP0_VM1"
    }
}

TASK [prepare_vmware_tests : Remove the test_vm* VMs] ********************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:13
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665 `" && echo ansible-tmp-1569865789.5738444-89606598796665="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpil9jkjbg TO /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665/ /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865789.5738444-89606598796665/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=test_vm1) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "test_vm1",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": "test_vm1"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893 `" && echo ansible-tmp-1569865790.2607253-157039777129893="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmptbffe347 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893/ /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865790.2607253-157039777129893/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=test_vm2) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "test_vm2",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": "test_vm2"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409 `" && echo ansible-tmp-1569865790.9607408-112312722750409="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpnlcfb26d TO /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409/ /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865790.9607408-112312722750409/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=test_vm3) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": [],
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "ha-datacenter",
            "datastore": null,
            "disk": [],
            "esxi_hostname": null,
            "folder": null,
            "force": true,
            "guest_id": null,
            "hardware": {},
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "test_vm3",
            "name_match": "first",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "absent",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": "test_vm3"
}

TASK [prepare_vmware_tests : Remove the DVSwitch] ************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:27
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259 `" && echo ansible-tmp-1569865791.843297-236634397168259="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_dvswitch.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp1n4fl4gc TO /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259/AnsiballZ_vmware_dvswitch.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259/ /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259/AnsiballZ_vmware_dvswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259/AnsiballZ_vmware_dvswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865791.843297-236634397168259/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=DVS0) => {
    "ansible_loop_var": "item",
    "changed": false,
    "dvswitch": "DVS0",
    "invocation": {
        "module_args": {
            "contact": null,
            "datacenter_name": "DC0",
            "description": null,
            "discovery_operation": "listen",
            "discovery_proto": "cdp",
            "folder": null,
            "health_check": {
                "teaming_failover": false,
                "teaming_failover_interval": 0,
                "vlan_mtu": false,
                "vlan_mtu_interval": 0
            },
            "hostname": "vcenter.test",
            "mtu": 1500,
            "multicast_filtering_mode": "basic",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "switch_name": "DVS0",
            "switch_version": null,
            "uplink_prefix": "Uplink ",
            "uplink_quantity": null,
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "DVS0",
    "result": "DVS not present"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615 `" && echo ansible-tmp-1569865792.5757844-89492758666615="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_dvswitch.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpy7dxhh30 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615/AnsiballZ_vmware_dvswitch.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615/ /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615/AnsiballZ_vmware_dvswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615/AnsiballZ_vmware_dvswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865792.5757844-89492758666615/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=dvswitch_0001) => {
    "ansible_loop_var": "item",
    "changed": false,
    "dvswitch": "dvswitch_0001",
    "invocation": {
        "module_args": {
            "contact": null,
            "datacenter_name": "DC0",
            "description": null,
            "discovery_operation": "listen",
            "discovery_proto": "cdp",
            "folder": null,
            "health_check": {
                "teaming_failover": false,
                "teaming_failover_interval": 0,
                "vlan_mtu": false,
                "vlan_mtu_interval": 0
            },
            "hostname": "vcenter.test",
            "mtu": 1500,
            "multicast_filtering_mode": "basic",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "switch_name": "dvswitch_0001",
            "switch_version": null,
            "uplink_prefix": "Uplink ",
            "uplink_quantity": null,
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "dvswitch_0001",
    "result": "DVS not present"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744 `" && echo ansible-tmp-1569865793.143361-190149908909744="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_dvswitch.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp14itb5yg TO /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744/AnsiballZ_vmware_dvswitch.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744/ /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744/AnsiballZ_vmware_dvswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744/AnsiballZ_vmware_dvswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865793.143361-190149908909744/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=dvswitch_0002) => {
    "ansible_loop_var": "item",
    "changed": false,
    "dvswitch": "dvswitch_0002",
    "invocation": {
        "module_args": {
            "contact": null,
            "datacenter_name": "DC0",
            "description": null,
            "discovery_operation": "listen",
            "discovery_proto": "cdp",
            "folder": null,
            "health_check": {
                "teaming_failover": false,
                "teaming_failover_interval": 0,
                "vlan_mtu": false,
                "vlan_mtu_interval": 0
            },
            "hostname": "vcenter.test",
            "mtu": 1500,
            "multicast_filtering_mode": "basic",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "switch_name": "dvswitch_0002",
            "switch_version": null,
            "uplink_prefix": "Uplink ",
            "uplink_quantity": null,
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "dvswitch_0002",
    "result": "DVS not present"
}

TASK [prepare_vmware_tests : Remove the vSwitches] ***********************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:42
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041 `" && echo ansible-tmp-1569865793.960335-132552786431041="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_vswitch.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp3133h_gp TO /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041/AnsiballZ_vmware_vswitch.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041/ /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041/AnsiballZ_vmware_vswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041/AnsiballZ_vmware_vswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865793.960335-132552786431041/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "esxi_hostname": null,
            "hostname": "esxi1.test",
            "mtu": 1500,
            "nics": [],
            "number_of_ports": 128,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "switch": "switch1",
            "switch_name": "switch1",
            "username": "root",
            "validate_certs": false
        }
    },
    "item": "esxi1.test"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337 `" && echo ansible-tmp-1569865794.6641643-73546625858337="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_vswitch.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpnizp0vgp TO /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337/AnsiballZ_vmware_vswitch.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337/ /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337/AnsiballZ_vmware_vswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337/AnsiballZ_vmware_vswitch.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865794.6641643-73546625858337/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "esxi_hostname": null,
            "hostname": "esxi2.test",
            "mtu": 1500,
            "nics": [],
            "number_of_ports": 128,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "switch": "switch1",
            "switch_name": "switch1",
            "username": "root",
            "validate_certs": false
        }
    },
    "item": "esxi2.test"
}

TASK [prepare_vmware_tests : Remove ESXi Hosts to vCenter] ***************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:53
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965 `" && echo ansible-tmp-1569865795.401875-89471107080965="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp_68gv89t TO /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965/AnsiballZ_vmware_host.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965/ /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865795.401875-89471107080965/ > /dev/null 2>&1 && sleep 0'
failed: [testhost] (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "add_connected": true,
            "cluster_name": "ccr1",
            "datacenter_name": "DC0",
            "esxi_hostname": "esxi1.test",
            "esxi_password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "esxi_ssl_thumbprint": "",
            "esxi_username": "root",
            "fetch_ssl_thumbprint": true,
            "folder": null,
            "force_connection": true,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "reconnect_disconnected": true,
            "state": "absent",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "esxi1.test",
    "msg": "Unable to find cluster with name ccr1"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564 `" && echo ansible-tmp-1569865796.1404188-21590836939564="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp4nfmac_0 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564/AnsiballZ_vmware_host.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564/ /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865796.1404188-21590836939564/ > /dev/null 2>&1 && sleep 0'
failed: [testhost] (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "add_connected": true,
            "cluster_name": "ccr1",
            "datacenter_name": "DC0",
            "esxi_hostname": "esxi2.test",
            "esxi_password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "esxi_ssl_thumbprint": "",
            "esxi_username": "root",
            "fetch_ssl_thumbprint": true,
            "folder": null,
            "force_connection": true,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "reconnect_disconnected": true,
            "state": "absent",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "esxi2.test",
    "msg": "Unable to find cluster with name ccr1"
}
...ignoring

TASK [prepare_vmware_tests : Umount NFS datastores to ESXi (1/2)] ********************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:68
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438 `" && echo ansible-tmp-1569865796.9280508-146194630229438="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpcpn9o66j TO /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438/ /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865796.9280508-146194630229438/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_0",
            "datastore_type": null,
            "esxi_hostname": "esxi1.test",
            "hostname": "esxi1.test",
            "nfs_path": null,
            "nfs_ro": false,
            "nfs_server": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi1.test",
    "result": "Datastore LocalDS_0 on host esxi1.test"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728 `" && echo ansible-tmp-1569865797.6726427-4690370288728="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp02qni0kk TO /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728/ /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865797.6726427-4690370288728/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_0",
            "datastore_type": null,
            "esxi_hostname": "esxi2.test",
            "hostname": "esxi2.test",
            "nfs_path": null,
            "nfs_ro": false,
            "nfs_server": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi2.test",
    "result": "Datastore LocalDS_0 on host esxi2.test"
}

TASK [prepare_vmware_tests : Umount NFS datastores to ESXi (2/2)] ********************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:79
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860 `" && echo ansible-tmp-1569865798.4903257-94862175389860="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp9jfqz28u TO /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860/ /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865798.4903257-94862175389860/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_1",
            "datastore_type": null,
            "esxi_hostname": "esxi1.test",
            "hostname": "esxi1.test",
            "nfs_path": null,
            "nfs_ro": false,
            "nfs_server": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi1.test",
    "result": "Datastore LocalDS_1 on host esxi1.test"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072 `" && echo ansible-tmp-1569865799.0744293-170130347267072="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmplh2e1e98 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072/ /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865799.0744293-170130347267072/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_1",
            "datastore_type": null,
            "esxi_hostname": "esxi2.test",
            "hostname": "esxi2.test",
            "nfs_path": null,
            "nfs_ro": false,
            "nfs_server": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi2.test",
    "result": "Datastore LocalDS_1 on host esxi2.test"
}

TASK [prepare_vmware_tests : Delete a datastore cluster to datacenter] ***************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:90
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860 `" && echo ansible-tmp-1569865799.9155529-21028312817860="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_datastore_cluster.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpx6lza8mc TO /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860/AnsiballZ_vmware_datastore_cluster.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860/ /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860/AnsiballZ_vmware_datastore_cluster.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860/AnsiballZ_vmware_datastore_cluster.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865799.9155529-21028312817860/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=DSC1) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "datacenter_name": "DC0",
            "datastore_cluster_name": "DSC1",
            "folder": null,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "DSC1",
    "result": "Datastore cluster 'DSC1' not available or already deleted."
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113 `" && echo ansible-tmp-1569865800.6151445-249626400185113="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_datastore_cluster.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpy9xbriae TO /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113/AnsiballZ_vmware_datastore_cluster.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113/ /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113/AnsiballZ_vmware_datastore_cluster.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113/AnsiballZ_vmware_datastore_cluster.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865800.6151445-249626400185113/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=DSC2) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "datacenter_name": "DC0",
            "datastore_cluster_name": "DSC2",
            "folder": null,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "DSC2",
    "result": "Datastore cluster 'DSC2' not available or already deleted."
}

TASK [prepare_vmware_tests : Remove the datacenter] **********************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:104
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371 `" && echo ansible-tmp-1569865801.4658368-95695555589371="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_datacenter.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp6njbzc_m TO /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371/AnsiballZ_vmware_datacenter.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371/ /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371/AnsiballZ_vmware_datacenter.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371/AnsiballZ_vmware_datacenter.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865801.4658368-95695555589371/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=DC0) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": "DC0",
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "DC0",
    "result": null
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138 `" && echo ansible-tmp-1569865803.733353-62720281133138="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_datacenter.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpfa14d809 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138/AnsiballZ_vmware_datacenter.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138/ /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138/AnsiballZ_vmware_datacenter.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138/AnsiballZ_vmware_datacenter.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865803.733353-62720281133138/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=datacenter_0001) => {
    "ansible_loop_var": "item",
    "changed": false,
    "invocation": {
        "module_args": {
            "datacenter_name": "datacenter_0001",
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "datacenter_0001"
}

TASK [prepare_vmware_tests : kill vcsim] *********************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/teardown.yml:117
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:10
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datacenter.yml for testhost

TASK [prepare_vmware_tests : Create Datacenter] **************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datacenter.yml:1
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585 `" && echo ansible-tmp-1569865805.1747067-230697366517585="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_datacenter.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmphdcxs09h TO /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585/AnsiballZ_vmware_datacenter.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585/ /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585/AnsiballZ_vmware_datacenter.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585/AnsiballZ_vmware_datacenter.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865805.1747067-230697366517585/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": "DC0",
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    }
}

TASK [prepare_vmware_tests : Create a VM folder on given Datacenter] *****************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datacenter.yml:10
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212 `" && echo ansible-tmp-1569865805.9935348-119606835038212="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vcenter_folder.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpwkdkoppd TO /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212/AnsiballZ_vcenter_folder.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212/ /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212/AnsiballZ_vcenter_folder.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212/AnsiballZ_vcenter_folder.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865805.9935348-119606835038212/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter": "DC0",
            "folder_name": "F0",
            "folder_type": "vm",
            "hostname": "vcenter.test",
            "parent_folder": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "result": {
        "msg": "Folder 'F0' of type 'vm' created successfully.",
        "path": "/DC0/vm/F0"
    }
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:11
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_cluster.yml for testhost

TASK [prepare_vmware_tests : Create Cluster] *****************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_cluster.yml:1
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933 `" && echo ansible-tmp-1569865807.403531-39186080848933="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_cluster.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp_uy62y1a TO /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933/AnsiballZ_vmware_cluster.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933/ /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933/AnsiballZ_vmware_cluster.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933/AnsiballZ_vmware_cluster.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865807.403531-39186080848933/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "cluster_name": "DC0_C0",
            "datacenter": "DC0",
            "datacenter_name": "DC0",
            "drs_default_vm_behavior": "fullyAutomated",
            "drs_enable_vm_behavior_overrides": true,
            "drs_vmotion_rate": 3,
            "enable_drs": false,
            "enable_ha": false,
            "enable_vsan": false,
            "ha_admission_control_enabled": true,
            "ha_failover_level": 2,
            "ha_host_monitoring": "enabled",
            "ha_restart_priority": "medium",
            "ha_vm_failure_interval": 30,
            "ha_vm_max_failure_window": -1,
            "ha_vm_max_failures": 3,
            "ha_vm_min_up_time": 120,
            "ha_vm_monitoring": "vmMonitoringDisabled",
            "hostname": "vcenter.test",
            "ignore_drs": false,
            "ignore_ha": false,
            "ignore_vsan": false,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "administrator@vsphere.local",
            "validate_certs": false,
            "vsan_auto_claim_storage": false
        }
    }
}

TASK [prepare_vmware_tests : Enable DRS on Cluster] **********************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_cluster.yml:10
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259 `" && echo ansible-tmp-1569865808.4687495-239885283493259="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_cluster_drs.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpadf2i5me TO /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259/AnsiballZ_vmware_cluster_drs.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259/ /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259/AnsiballZ_vmware_cluster_drs.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259/AnsiballZ_vmware_cluster_drs.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865808.4687495-239885283493259/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "cluster_name": "DC0_C0",
            "datacenter": "DC0",
            "datacenter_name": "DC0",
            "drs_default_vm_behavior": "fullyAutomated",
            "drs_enable_vm_behavior_overrides": true,
            "drs_vmotion_rate": 3,
            "enable_drs": true,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "result": null
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:12
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_attach_hosts.yml for testhost

TASK [prepare_vmware_tests : Add ESXi Hosts to vCenter] ******************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_attach_hosts.yml:1
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638 `" && echo ansible-tmp-1569865811.1598444-124314093360638="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpr8wcb88z TO /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638/AnsiballZ_vmware_host.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638/ /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865811.1598444-124314093360638/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "add_connected": true,
            "cluster_name": "DC0_C0",
            "datacenter_name": "DC0",
            "esxi_hostname": "esxi1.test",
            "esxi_password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "esxi_ssl_thumbprint": "",
            "esxi_username": "root",
            "fetch_ssl_thumbprint": true,
            "folder": null,
            "force_connection": true,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "reconnect_disconnected": true,
            "state": "present",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "esxi1.test",
    "result": "Host connected to vCenter 'vcenter.test'"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571 `" && echo ansible-tmp-1569865819.7200801-108874258378571="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp_uallsns TO /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571/AnsiballZ_vmware_host.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571/ /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571/AnsiballZ_vmware_host.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865819.7200801-108874258378571/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "add_connected": true,
            "cluster_name": "DC0_C0",
            "datacenter_name": "DC0",
            "esxi_hostname": "esxi2.test",
            "esxi_password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "esxi_ssl_thumbprint": "",
            "esxi_username": "root",
            "fetch_ssl_thumbprint": true,
            "folder": null,
            "force_connection": true,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "reconnect_disconnected": true,
            "state": "present",
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    },
    "item": "esxi2.test",
    "result": "Host connected to vCenter 'vcenter.test'"
}

TASK [prepare_vmware_tests : Disable the Maintenance Mode] ***************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_attach_hosts.yml:15
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806 `" && echo ansible-tmp-1569865829.2952657-10190837811806="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_maintenancemode.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp7zl0lnkb TO /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806/AnsiballZ_vmware_maintenancemode.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806/ /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806/AnsiballZ_vmware_maintenancemode.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806/AnsiballZ_vmware_maintenancemode.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865829.2952657-10190837811806/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": false,
    "hostname": "esxi1.test",
    "hostsystem": "'vim.HostSystem:host-51'",
    "invocation": {
        "module_args": {
            "esxi_hostname": "esxi1.test",
            "evacuate": false,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "timeout": 0,
            "username": "administrator@vsphere.local",
            "validate_certs": false,
            "vsan": null
        }
    },
    "item": "esxi1.test",
    "msg": "Host esxi1.test not in maintenance mode",
    "status": "NO_ACTION"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726 `" && echo ansible-tmp-1569865830.1354983-250181187066726="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_maintenancemode.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpbe3wxacm TO /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726/AnsiballZ_vmware_maintenancemode.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726/ /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726/AnsiballZ_vmware_maintenancemode.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726/AnsiballZ_vmware_maintenancemode.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865830.1354983-250181187066726/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": false,
    "hostname": "esxi2.test",
    "hostsystem": "'vim.HostSystem:host-54'",
    "invocation": {
        "module_args": {
            "esxi_hostname": "esxi2.test",
            "evacuate": false,
            "hostname": "vcenter.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "absent",
            "timeout": 0,
            "username": "administrator@vsphere.local",
            "validate_certs": false,
            "vsan": null
        }
    },
    "item": "esxi2.test",
    "msg": "Host esxi2.test not in maintenance mode",
    "status": "NO_ACTION"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:14
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datastore.yml for testhost

TASK [prepare_vmware_tests : Mount NFS (ds1) datastores to ESXi] *********************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datastore.yml:2
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679 `" && echo ansible-tmp-1569865831.3232305-231936385421679="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpm0piacc7 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679/ /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865831.3232305-231936385421679/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_0",
            "datastore_type": "nfs",
            "esxi_hostname": null,
            "hostname": "esxi1.test",
            "nfs_path": "/srv/share/isos",
            "nfs_ro": false,
            "nfs_server": "gateway.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi1.test",
    "result": "Datastore LocalDS_0 on host esxi1.test"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947 `" && echo ansible-tmp-1569865831.9841492-85148647339947="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpw96nbx_0 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947/ /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865831.9841492-85148647339947/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_0",
            "datastore_type": "nfs",
            "esxi_hostname": null,
            "hostname": "esxi2.test",
            "nfs_path": "/srv/share/isos",
            "nfs_ro": false,
            "nfs_server": "gateway.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi2.test",
    "result": "Datastore LocalDS_0 on host esxi2.test"
}

TASK [prepare_vmware_tests : Mount NFS (ds2) datastores on the ESXi] *****************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datastore.yml:16
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259 `" && echo ansible-tmp-1569865832.8422904-225431450853259="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmphcyh71ct TO /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259/ /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865832.8422904-225431450853259/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi1.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_1",
            "datastore_type": "nfs",
            "esxi_hostname": null,
            "hostname": "esxi1.test",
            "nfs_path": "/srv/share/vms",
            "nfs_ro": false,
            "nfs_server": "gateway.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi1.test",
    "result": "Datastore LocalDS_1 on host esxi1.test"
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604 `" && echo ansible-tmp-1569865833.4066527-121604383256604="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_host_datastore.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpzmifu649 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604/AnsiballZ_vmware_host_datastore.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604/ /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604/AnsiballZ_vmware_host_datastore.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865833.4066527-121604383256604/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item=esxi2.test) => {
    "ansible_loop_var": "item",
    "changed": true,
    "invocation": {
        "module_args": {
            "datacenter_name": null,
            "datastore_name": "LocalDS_1",
            "datastore_type": "nfs",
            "esxi_hostname": null,
            "hostname": "esxi2.test",
            "nfs_path": "/srv/share/vms",
            "nfs_ro": false,
            "nfs_server": "gateway.test",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "state": "present",
            "username": "root",
            "validate_certs": false,
            "vmfs_device_name": null,
            "vmfs_version": null
        }
    },
    "item": "esxi2.test",
    "result": "Datastore LocalDS_1 on host esxi2.test"
}

TASK [prepare_vmware_tests : The vcenter needs a bit of time to refresh the DS list] *************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_datastore.yml:30
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782 `" && echo ansible-tmp-1569865834.337931-162334687146782="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_datastore_info.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpe5m3_cqy TO /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782/AnsiballZ_vmware_datastore_info.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782/ /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782/AnsiballZ_vmware_datastore_info.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782/AnsiballZ_vmware_datastore_info.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865834.337931-162334687146782/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => {
    "attempts": 1,
    "changed": false,
    "datastores": [
        {
            "accessible": true,
            "capacity": 2684354560,
            "datastore_cluster": "N/A",
            "freeSpace": 1175453696,
            "maintenanceMode": "normal",
            "multipleHostAccess": false,
            "name": "datastore1 (1)",
            "provisioned": 1508900864,
            "type": "VMFS",
            "uncommitted": 0,
            "url": "ds:///vmfs/volumes/5d8cf1ef-41d608cc-08a6-525400e619d0/"
        },
        {
            "accessible": true,
            "capacity": 493718847488,
            "datastore_cluster": "N/A",
            "freeSpace": 67920011264,
            "maintenanceMode": "normal",
            "multipleHostAccess": false,
            "name": "LocalDS_1",
            "provisioned": 425798836224,
            "type": "NFS",
            "uncommitted": 0,
            "url": "ds:///vmfs/volumes/c0bbef61-d5da798c/"
        },
        {
            "accessible": true,
            "capacity": 493718847488,
            "datastore_cluster": "N/A",
            "freeSpace": 67919618048,
            "maintenanceMode": "normal",
            "multipleHostAccess": false,
            "name": "LocalDS_0",
            "provisioned": 425799229440,
            "type": "NFS",
            "uncommitted": 0,
            "url": "ds:///vmfs/volumes/c167bb72-db7159d1/"
        },
        {
            "accessible": true,
            "capacity": 2684354560,
            "datastore_cluster": "N/A",
            "freeSpace": 1175453696,
            "maintenanceMode": "normal",
            "multipleHostAccess": false,
            "name": "datastore1",
            "provisioned": 1508900864,
            "type": "VMFS",
            "uncommitted": 0,
            "url": "ds:///vmfs/volumes/5d8cf11c-e8aaee62-5a61-525400341bb1/"
        }
    ],
    "failed_when_result": false,
    "invocation": {
        "module_args": {
            "cluster": "DC0_C0",
            "datacenter": null,
            "gather_nfs_mount_info": false,
            "gather_vmfs_mount_info": false,
            "hostname": "vcenter.test",
            "name": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "username": "administrator@vsphere.local",
            "validate_certs": false
        }
    }
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:16
included: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_virtualmachines.yml for testhost

TASK [prepare_vmware_tests : Create VMs] *********************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_virtualmachines.yml:2
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212 `" && echo ansible-tmp-1569865836.2448084-159593909991212="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmps4v_ropx TO /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212/ /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865836.2448084-159593909991212/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_H0_VM0', 'folder': 'F0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "instance": {
        "annotation": "",
        "current_snapshot": null,
        "customvalues": {},
        "guest_consolidation_needed": false,
        "guest_question": null,
        "guest_tools_status": "guestToolsNotRunning",
        "guest_tools_version": "0",
        "hw_cluster": "DC0_C0",
        "hw_cores_per_socket": 1,
        "hw_datastores": [
            "LocalDS_1",
            "LocalDS_0"
        ],
        "hw_esxi_host": "esxi2.test",
        "hw_eth0": {
            "addresstype": "assigned",
            "ipaddresses": null,
            "label": "Network adapter 1",
            "macaddress": "00:50:56:82:c7:fb",
            "macaddress_dash": "00-50-56-82-c7-fb",
            "portgroup_key": null,
            "portgroup_portkey": null,
            "summary": "VM Network"
        },
        "hw_files": [
            "[LocalDS_1] DC0_H0_VM0_3/DC0_H0_VM0.vmx",
            "[LocalDS_1] DC0_H0_VM0_3/DC0_H0_VM0.vmsd",
            "[LocalDS_1] DC0_H0_VM0_3/DC0_H0_VM0.vmdk"
        ],
        "hw_folder": "/DC0/vm/F0",
        "hw_guest_full_name": null,
        "hw_guest_ha_state": null,
        "hw_guest_id": null,
        "hw_interfaces": [
            "eth0"
        ],
        "hw_is_template": false,
        "hw_memtotal_mb": 128,
        "hw_name": "DC0_H0_VM0",
        "hw_power_status": "poweredOff",
        "hw_processor_count": 1,
        "hw_product_uuid": "420294ed-c147-967d-cdab-da097431df14",
        "hw_version": "vmx-14",
        "instance_uuid": "50026a0e-f968-0003-9f9e-15a1014a7272",
        "ipv4": null,
        "ipv6": null,
        "module_hw": true,
        "moid": "vm-58",
        "snapshots": [],
        "vimref": "vim.VirtualMachine:vm-58",
        "vnc": {}
    },
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": {
                "iso_path": "[LocalDS_0] fedora.iso",
                "type": "iso"
            },
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "DC0",
            "datastore": null,
            "disk": [
                {
                    "datastore": "LocalDS_1",
                    "size_gb": 1,
                    "type": "thin"
                }
            ],
            "esxi_hostname": null,
            "folder": "F0",
            "force": false,
            "guest_id": "debian8_64Guest",
            "hardware": {
                "memory_mb": 128,
                "num_cpus": 1,
                "scsi": "paravirtual"
            },
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_H0_VM0",
            "name_match": "first",
            "networks": [
                {
                    "name": "VM Network",
                    "type": "dhcp"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "poweredoff",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "folder": "F0",
        "name": "DC0_H0_VM0"
    }
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488 `" && echo ansible-tmp-1569865838.2405634-39748147757488="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpay100_3e TO /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488/ /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865838.2405634-39748147757488/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_H0_VM1', 'folder': 'F0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "instance": {
        "annotation": "",
        "current_snapshot": null,
        "customvalues": {},
        "guest_consolidation_needed": false,
        "guest_question": null,
        "guest_tools_status": "guestToolsNotRunning",
        "guest_tools_version": "0",
        "hw_cluster": "DC0_C0",
        "hw_cores_per_socket": 1,
        "hw_datastores": [
            "LocalDS_1",
            "LocalDS_0"
        ],
        "hw_esxi_host": "esxi1.test",
        "hw_eth0": {
            "addresstype": "assigned",
            "ipaddresses": null,
            "label": "Network adapter 1",
            "macaddress": "00:50:56:82:08:ea",
            "macaddress_dash": "00-50-56-82-08-ea",
            "portgroup_key": null,
            "portgroup_portkey": null,
            "summary": "VM Network"
        },
        "hw_files": [
            "[LocalDS_1] DC0_H0_VM1_2/DC0_H0_VM1.vmx",
            "[LocalDS_1] DC0_H0_VM1_2/DC0_H0_VM1.vmsd",
            "[LocalDS_1] DC0_H0_VM1_2/DC0_H0_VM1.vmdk"
        ],
        "hw_folder": "/DC0/vm/F0",
        "hw_guest_full_name": null,
        "hw_guest_ha_state": null,
        "hw_guest_id": null,
        "hw_interfaces": [
            "eth0"
        ],
        "hw_is_template": false,
        "hw_memtotal_mb": 128,
        "hw_name": "DC0_H0_VM1",
        "hw_power_status": "poweredOff",
        "hw_processor_count": 1,
        "hw_product_uuid": "42026b7e-d053-8956-85e4-88f36d69daf7",
        "hw_version": "vmx-14",
        "instance_uuid": "5002a420-ec4e-16e3-9656-4e08dc097028",
        "ipv4": null,
        "ipv6": null,
        "module_hw": true,
        "moid": "vm-59",
        "snapshots": [],
        "vimref": "vim.VirtualMachine:vm-59",
        "vnc": {}
    },
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": {
                "iso_path": "[LocalDS_0] fedora.iso",
                "type": "iso"
            },
            "cluster": null,
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "DC0",
            "datastore": null,
            "disk": [
                {
                    "datastore": "LocalDS_1",
                    "size_gb": 1,
                    "type": "thin"
                }
            ],
            "esxi_hostname": null,
            "folder": "F0",
            "force": false,
            "guest_id": "debian8_64Guest",
            "hardware": {
                "memory_mb": 128,
                "num_cpus": 1,
                "scsi": "paravirtual"
            },
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_H0_VM1",
            "name_match": "first",
            "networks": [
                {
                    "name": "VM Network",
                    "type": "dhcp"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "poweredoff",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "folder": "F0",
        "name": "DC0_H0_VM1"
    }
}

TASK [prepare_vmware_tests : Create VMs in cluster] **********************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/setup_virtualmachines.yml:29
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313 `" && echo ansible-tmp-1569865840.4740443-142221418046313="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp0t33m6vr TO /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313/ /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865840.4740443-142221418046313/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_C0_RP0_VM0', 'folder': 'F0', 'cluster': 'DC0_C0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "instance": {
        "annotation": "",
        "current_snapshot": null,
        "customvalues": {},
        "guest_consolidation_needed": false,
        "guest_question": null,
        "guest_tools_status": "guestToolsNotRunning",
        "guest_tools_version": "0",
        "hw_cluster": "DC0_C0",
        "hw_cores_per_socket": 1,
        "hw_datastores": [
            "LocalDS_1",
            "LocalDS_0"
        ],
        "hw_esxi_host": "esxi2.test",
        "hw_eth0": {
            "addresstype": "assigned",
            "ipaddresses": null,
            "label": "Network adapter 1",
            "macaddress": "00:50:56:82:a7:11",
            "macaddress_dash": "00-50-56-82-a7-11",
            "portgroup_key": null,
            "portgroup_portkey": null,
            "summary": "VM Network"
        },
        "hw_files": [
            "[LocalDS_1] DC0_C0_RP0_VM0_2/DC0_C0_RP0_VM0.vmx",
            "[LocalDS_1] DC0_C0_RP0_VM0_2/DC0_C0_RP0_VM0.vmsd",
            "[LocalDS_1] DC0_C0_RP0_VM0_2/DC0_C0_RP0_VM0.vmdk"
        ],
        "hw_folder": "/DC0/vm/F0",
        "hw_guest_full_name": null,
        "hw_guest_ha_state": null,
        "hw_guest_id": null,
        "hw_interfaces": [
            "eth0"
        ],
        "hw_is_template": false,
        "hw_memtotal_mb": 128,
        "hw_name": "DC0_C0_RP0_VM0",
        "hw_power_status": "poweredOff",
        "hw_processor_count": 1,
        "hw_product_uuid": "4202414b-00ee-59a7-63ba-a5fbe68bb722",
        "hw_version": "vmx-14",
        "instance_uuid": "50022022-0a33-19fe-5b36-eb8692f0b295",
        "ipv4": null,
        "ipv6": null,
        "module_hw": true,
        "moid": "vm-60",
        "snapshots": [],
        "vimref": "vim.VirtualMachine:vm-60",
        "vnc": {}
    },
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": {
                "iso_path": "[LocalDS_0] fedora.iso",
                "type": "iso"
            },
            "cluster": "DC0_C0",
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "DC0",
            "datastore": null,
            "disk": [
                {
                    "datastore": "LocalDS_1",
                    "size_gb": 1,
                    "type": "thin"
                }
            ],
            "esxi_hostname": null,
            "folder": "F0",
            "force": false,
            "guest_id": "debian8_64Guest",
            "hardware": {
                "memory_mb": 128,
                "num_cpus": 1,
                "scsi": "paravirtual"
            },
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_C0_RP0_VM0",
            "name_match": "first",
            "networks": [
                {
                    "name": "VM Network",
                    "type": "dhcp"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "poweredoff",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "cluster": "DC0_C0",
        "folder": "F0",
        "name": "DC0_C0_RP0_VM0"
    }
}
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887 `" && echo ansible-tmp-1569865842.484583-112571332070887="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpscrybb2z TO /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887/AnsiballZ_vmware_guest.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887/ /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887/AnsiballZ_vmware_guest.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865842.484583-112571332070887/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => (item={'name': 'DC0_C0_RP0_VM1', 'folder': 'F0', 'cluster': 'DC0_C0'}) => {
    "ansible_loop_var": "item",
    "changed": true,
    "instance": {
        "annotation": "",
        "current_snapshot": null,
        "customvalues": {},
        "guest_consolidation_needed": false,
        "guest_question": null,
        "guest_tools_status": "guestToolsNotRunning",
        "guest_tools_version": "0",
        "hw_cluster": "DC0_C0",
        "hw_cores_per_socket": 1,
        "hw_datastores": [
            "LocalDS_1",
            "LocalDS_0"
        ],
        "hw_esxi_host": "esxi2.test",
        "hw_eth0": {
            "addresstype": "assigned",
            "ipaddresses": null,
            "label": "Network adapter 1",
            "macaddress": "00:50:56:82:a3:d5",
            "macaddress_dash": "00-50-56-82-a3-d5",
            "portgroup_key": null,
            "portgroup_portkey": null,
            "summary": "VM Network"
        },
        "hw_files": [
            "[LocalDS_1] DC0_C0_RP0_VM1_2/DC0_C0_RP0_VM1.vmx",
            "[LocalDS_1] DC0_C0_RP0_VM1_2/DC0_C0_RP0_VM1.vmsd",
            "[LocalDS_1] DC0_C0_RP0_VM1_2/DC0_C0_RP0_VM1.vmdk"
        ],
        "hw_folder": "/DC0/vm/F0",
        "hw_guest_full_name": null,
        "hw_guest_ha_state": null,
        "hw_guest_id": null,
        "hw_interfaces": [
            "eth0"
        ],
        "hw_is_template": false,
        "hw_memtotal_mb": 128,
        "hw_name": "DC0_C0_RP0_VM1",
        "hw_power_status": "poweredOff",
        "hw_processor_count": 1,
        "hw_product_uuid": "420222be-cfc8-cd73-f759-b430dde34542",
        "hw_version": "vmx-14",
        "instance_uuid": "50023b5b-e44e-70b6-f548-db4241798878",
        "ipv4": null,
        "ipv6": null,
        "module_hw": true,
        "moid": "vm-61",
        "snapshots": [],
        "vimref": "vim.VirtualMachine:vm-61",
        "vnc": {}
    },
    "invocation": {
        "module_args": {
            "annotation": null,
            "cdrom": {
                "iso_path": "[LocalDS_0] fedora.iso",
                "type": "iso"
            },
            "cluster": "DC0_C0",
            "convert": null,
            "customization": {},
            "customization_spec": null,
            "customvalues": [],
            "datacenter": "DC0",
            "datastore": null,
            "disk": [
                {
                    "datastore": "LocalDS_1",
                    "size_gb": 1,
                    "type": "thin"
                }
            ],
            "esxi_hostname": null,
            "folder": "F0",
            "force": false,
            "guest_id": "debian8_64Guest",
            "hardware": {
                "memory_mb": 128,
                "num_cpus": 1,
                "scsi": "paravirtual"
            },
            "hostname": "vcenter.test",
            "is_template": false,
            "linked_clone": false,
            "name": "DC0_C0_RP0_VM1",
            "name_match": "first",
            "networks": [
                {
                    "name": "VM Network",
                    "type": "dhcp"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "resource_pool": null,
            "snapshot_src": null,
            "state": "poweredoff",
            "state_change_timeout": 0,
            "template": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false,
            "vapp_properties": [],
            "wait_for_customization": false,
            "wait_for_ip_address": false
        }
    },
    "item": {
        "cluster": "DC0_C0",
        "folder": "F0",
        "name": "DC0_C0_RP0_VM1"
    }
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:18
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:20
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:22
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:24
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:26
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml:28
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [prepare_vmware_tests : include_tasks] ******************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/prepare_vmware_tests/tasks/main.yml:10
skipping: [testhost] => {
    "changed": false,
    "skip_reason": "Conditional result was False"
}

TASK [vmware_guest_network : gather network adapters' facts of the virtual machine] **************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:14
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295 `" && echo ansible-tmp-1569865846.283572-153030365464295="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_network.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpzg6hz8pa TO /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295/AnsiballZ_vmware_guest_network.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295/ /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865846.283572-153030365464295/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => {
    "changed": false,
    "invocation": {
        "module_args": {
            "cluster": null,
            "datacenter": "ha-datacenter",
            "esxi_hostname": null,
            "folder": null,
            "gather_network_info": true,
            "hostname": "vcenter.test",
            "moid": null,
            "name": "DC0_H0_VM0",
            "networks": [],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    },
    "network_data": {
        "0": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 1",
            "mac_addr": "00:50:56:82:c7:fb",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 7,
            "wake_onlan": true
        }
    }
}

TASK [vmware_guest_network : debug] **************************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:24
ok: [testhost] => {
    "netadapter_info": {
        "changed": false,
        "failed": false,
        "network_data": {
            "0": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 1",
                "mac_addr": "00:50:56:82:c7:fb",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 7,
                "wake_onlan": true
            }
        }
    }
}

TASK [vmware_guest_network : get number of existing netowrk adapters] ****************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:26
ok: [testhost] => {
    "ansible_facts": {
        "netadapter_num": "1"
    },
    "changed": false
}

TASK [vmware_guest_network : add new network adapters to virtual machine] ************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:30
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580 `" && echo ansible-tmp-1569865847.7829936-249180473003580="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_network.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpisfpjkyk TO /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580/AnsiballZ_vmware_guest_network.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580/ /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865847.7829936-249180473003580/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "cluster": null,
            "datacenter": "ha-datacenter",
            "esxi_hostname": null,
            "folder": null,
            "gather_network_info": false,
            "hostname": "vcenter.test",
            "moid": null,
            "name": "DC0_H0_VM0",
            "networks": [
                {
                    "device_type": "e1000e",
                    "manual_mac": "00:50:56:58:59:60",
                    "name": "VM Network",
                    "state": "new"
                },
                {
                    "device_type": "vmxnet3",
                    "manual_mac": "00:50:56:58:59:61",
                    "name": "VM Network",
                    "state": "new"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    },
    "network_data": {
        "0": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 1",
            "mac_addr": "00:50:56:82:c7:fb",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 7,
            "wake_onlan": true
        },
        "1": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "E1000E",
            "label": "Network adapter 2",
            "mac_addr": "00:50:56:58:59:60",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 8,
            "wake_onlan": true
        },
        "2": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 3",
            "mac_addr": "00:50:56:58:59:61",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 9,
            "wake_onlan": true
        }
    }
}

TASK [vmware_guest_network : debug] **************************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:48
ok: [testhost] => {
    "add_netadapter": {
        "changed": true,
        "failed": false,
        "network_data": {
            "0": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 1",
                "mac_addr": "00:50:56:82:c7:fb",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 7,
                "wake_onlan": true
            },
            "1": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "E1000E",
                "label": "Network adapter 2",
                "mac_addr": "00:50:56:58:59:60",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 8,
                "wake_onlan": true
            },
            "2": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 3",
                "mac_addr": "00:50:56:58:59:61",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 9,
                "wake_onlan": true
            }
        }
    }
}

TASK [vmware_guest_network : assert the new netowrk adapters were added to VM] *******************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:50
ok: [testhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [vmware_guest_network : delete one specified network adapter] *******************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:56
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813 `" && echo ansible-tmp-1569865851.2382033-188156699109813="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_network.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp6wsqzqss TO /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813/AnsiballZ_vmware_guest_network.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813/ /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865851.2382033-188156699109813/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "cluster": null,
            "datacenter": "ha-datacenter",
            "esxi_hostname": null,
            "folder": null,
            "gather_network_info": false,
            "hostname": "vcenter.test",
            "moid": null,
            "name": "DC0_H0_VM0",
            "networks": [
                {
                    "mac": "00:50:56:58:59:60",
                    "state": "absent"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    },
    "network_data": {
        "0": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 1",
            "mac_addr": "00:50:56:82:c7:fb",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 7,
            "wake_onlan": true
        },
        "1": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 3",
            "mac_addr": "00:50:56:58:59:61",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 9,
            "wake_onlan": true
        }
    }
}

TASK [vmware_guest_network : debug] **************************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:68
ok: [testhost] => {
    "del_netadapter": {
        "changed": true,
        "failed": false,
        "network_data": {
            "0": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 1",
                "mac_addr": "00:50:56:82:c7:fb",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 7,
                "wake_onlan": true
            },
            "1": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 3",
                "mac_addr": "00:50:56:58:59:61",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 9,
                "wake_onlan": true
            }
        }
    }
}

TASK [vmware_guest_network : assert the network adapter was removed] *****************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:70
ok: [testhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [vmware_guest_network : get instance uuid of virtual machines] ******************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:76
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956 `" && echo ansible-tmp-1569865854.6559446-13041409540956="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_info.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpnkov6ggq TO /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956/AnsiballZ_vmware_guest_info.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956/ /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956/AnsiballZ_vmware_guest_info.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956/AnsiballZ_vmware_guest_info.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865854.6559446-13041409540956/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => {
    "changed": false,
    "instance": {
        "annotation": "",
        "current_snapshot": null,
        "customvalues": {},
        "guest_consolidation_needed": false,
        "guest_question": null,
        "guest_tools_status": "guestToolsNotRunning",
        "guest_tools_version": "0",
        "hw_cluster": "DC0_C0",
        "hw_cores_per_socket": 1,
        "hw_datastores": [
            "LocalDS_1",
            "LocalDS_0"
        ],
        "hw_esxi_host": "esxi2.test",
        "hw_eth0": {
            "addresstype": "assigned",
            "ipaddresses": null,
            "label": "Network adapter 1",
            "macaddress": "00:50:56:82:c7:fb",
            "macaddress_dash": "00-50-56-82-c7-fb",
            "portgroup_key": null,
            "portgroup_portkey": null,
            "summary": "VM Network"
        },
        "hw_eth1": {
            "addresstype": "manual",
            "ipaddresses": null,
            "label": "Network adapter 3",
            "macaddress": "00:50:56:58:59:61",
            "macaddress_dash": "00-50-56-58-59-61",
            "portgroup_key": null,
            "portgroup_portkey": null,
            "summary": "VM Network"
        },
        "hw_files": [
            "[LocalDS_1] DC0_H0_VM0_3/DC0_H0_VM0.vmx",
            "[LocalDS_1] DC0_H0_VM0_3/DC0_H0_VM0.vmsd",
            "[LocalDS_1] DC0_H0_VM0_3/DC0_H0_VM0.vmdk"
        ],
        "hw_folder": "/DC0/vm/F0",
        "hw_guest_full_name": null,
        "hw_guest_ha_state": null,
        "hw_guest_id": null,
        "hw_interfaces": [
            "eth0",
            "eth1"
        ],
        "hw_is_template": false,
        "hw_memtotal_mb": 128,
        "hw_name": "DC0_H0_VM0",
        "hw_power_status": "poweredOff",
        "hw_processor_count": 1,
        "hw_product_uuid": "420294ed-c147-967d-cdab-da097431df14",
        "hw_version": "vmx-14",
        "instance_uuid": "50026a0e-f968-0003-9f9e-15a1014a7272",
        "ipv4": null,
        "ipv6": null,
        "module_hw": true,
        "moid": "vm-58",
        "snapshots": [],
        "vimref": "vim.VirtualMachine:vm-58",
        "vnc": {}
    },
    "invocation": {
        "module_args": {
            "datacenter": "DC0",
            "folder": null,
            "hostname": "vcenter.test",
            "moid": null,
            "name": "DC0_H0_VM0",
            "name_match": "first",
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "properties": null,
            "proxy_host": null,
            "proxy_port": null,
            "schema": "summary",
            "tags": false,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    }
}

TASK [vmware_guest_network : set_fact] ***********************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:86
ok: [testhost] => {
    "ansible_facts": {
        "vm1_instance_uuid": "50026a0e-f968-0003-9f9e-15a1014a7272"
    },
    "changed": false
}

TASK [vmware_guest_network : add new network adapters to virtual machine with instance uuid] *****************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:88
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657 `" && echo ansible-tmp-1569865856.5178556-211502768938657="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_network.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp70wpmw3_ TO /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657/AnsiballZ_vmware_guest_network.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657/ /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865856.5178556-211502768938657/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "cluster": null,
            "datacenter": "ha-datacenter",
            "esxi_hostname": null,
            "folder": null,
            "gather_network_info": false,
            "hostname": "vcenter.test",
            "moid": null,
            "name": null,
            "networks": [
                {
                    "device_type": "e1000e",
                    "manual_mac": "00:50:56:58:59:60",
                    "name": "VM Network",
                    "state": "new"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "use_instance_uuid": true,
            "username": "administrator@vsphere.local",
            "uuid": "50026a0e-f968-0003-9f9e-15a1014a7272",
            "validate_certs": false
        }
    },
    "network_data": {
        "0": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 1",
            "mac_addr": "00:50:56:82:c7:fb",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 7,
            "wake_onlan": true
        },
        "1": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 3",
            "mac_addr": "00:50:56:58:59:61",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 9,
            "wake_onlan": true
        },
        "2": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "E1000E",
            "label": "Network adapter 2",
            "mac_addr": "00:50:56:58:59:60",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 8,
            "wake_onlan": true
        }
    }
}

TASK [vmware_guest_network : debug] **************************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:103
ok: [testhost] => {
    "add_netadapter_instanceuuid": {
        "changed": true,
        "failed": false,
        "network_data": {
            "0": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 1",
                "mac_addr": "00:50:56:82:c7:fb",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 7,
                "wake_onlan": true
            },
            "1": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 3",
                "mac_addr": "00:50:56:58:59:61",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 9,
                "wake_onlan": true
            },
            "2": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "E1000E",
                "label": "Network adapter 2",
                "mac_addr": "00:50:56:58:59:60",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 8,
                "wake_onlan": true
            }
        }
    }
}

TASK [vmware_guest_network : assert the new netowrk adapters were added to VM] *******************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:105
ok: [testhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [vmware_guest_network : delete again one specified network adapter] *************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:111
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737 `" && echo ansible-tmp-1569865859.468401-274072543687737="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_network.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmp6jh4u9a3 TO /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737/AnsiballZ_vmware_guest_network.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737/ /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865859.468401-274072543687737/ > /dev/null 2>&1 && sleep 0'
changed: [testhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "cluster": null,
            "datacenter": "ha-datacenter",
            "esxi_hostname": null,
            "folder": null,
            "gather_network_info": false,
            "hostname": "vcenter.test",
            "moid": null,
            "name": "DC0_H0_VM0",
            "networks": [
                {
                    "mac": "00:50:56:58:59:60",
                    "state": "absent"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    },
    "network_data": {
        "0": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 1",
            "mac_addr": "00:50:56:82:c7:fb",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 7,
            "wake_onlan": true
        },
        "1": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 3",
            "mac_addr": "00:50:56:58:59:61",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 9,
            "wake_onlan": true
        }
    }
}

TASK [vmware_guest_network : debug] **************************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:123
ok: [testhost] => {
    "del_again_netadapter": {
        "changed": true,
        "failed": false,
        "network_data": {
            "0": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 1",
                "mac_addr": "00:50:56:82:c7:fb",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 7,
                "wake_onlan": true
            },
            "1": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 3",
                "mac_addr": "00:50:56:58:59:61",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 9,
                "wake_onlan": true
            }
        }
    }
}

TASK [vmware_guest_network : assert the network adapter was removed] *****************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:125
ok: [testhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [vmware_guest_network : disconnect one specified network adapter] ***************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:131
<testhost> ESTABLISH LOCAL CONNECTION FOR USER: goneri
<testhost> EXEC /bin/sh -c 'echo ~goneri && sleep 0'
<testhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633 `" && echo ansible-tmp-1569865862.8991458-190973922467633="` echo /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633 `" ) && sleep 0'
Using module file /home/goneri/git_repos/ansible_projects/a/lib/ansible/modules/cloud/vmware/vmware_guest_network.py
<testhost> PUT /home/goneri/.ansible/tmp/ansible-local-264429tmq9mmf/tmpjfkr6nyl TO /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633/AnsiballZ_vmware_guest_network.py
<testhost> EXEC /bin/sh -c 'chmod u+x /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633/ /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'ESXI1_HOSTNAME=esxi1.test ESXI1_PASSWORD='"'"'!234AaAa56'"'"' ESXI1_USERNAME=root ESXI2_HOSTNAME=test2.test ESXI2_PASSWORD='"'"'!234AaAa56'"'"' ESXI2_USERNAME=root VCENTER_HOSTNAME=vcenter.test VCENTER_PASSWORD='"'"'!234AaAa56'"'"' VCENTER_USERNAME=administrator@vsphere.local VMWARE_VALIDATE_CERTS=false /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633/AnsiballZ_vmware_guest_network.py && sleep 0'
<testhost> EXEC /bin/sh -c 'rm -f -r /home/goneri/.ansible/tmp/ansible-tmp-1569865862.8991458-190973922467633/ > /dev/null 2>&1 && sleep 0'
ok: [testhost] => {
    "changed": false,
    "invocation": {
        "module_args": {
            "cluster": null,
            "datacenter": "ha-datacenter",
            "esxi_hostname": null,
            "folder": null,
            "gather_network_info": false,
            "hostname": "vcenter.test",
            "moid": null,
            "name": "DC0_H0_VM0",
            "networks": [
                {
                    "connected": false,
                    "mac": "00:50:56:58:59:61",
                    "state": "present"
                }
            ],
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "use_instance_uuid": false,
            "username": "administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    },
    "network_data": {
        "0": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 1",
            "mac_addr": "00:50:56:82:c7:fb",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 7,
            "wake_onlan": true
        },
        "1": {
            "allow_guest_ctl": true,
            "connected": false,
            "device_type": "VMXNET3",
            "label": "Network adapter 3",
            "mac_addr": "00:50:56:58:59:61",
            "name": "VM Network",
            "start_connected": true,
            "unit_number": 9,
            "wake_onlan": true
        }
    }
}

TASK [vmware_guest_network : debug] **************************************************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:144
ok: [testhost] => {
    "disc_netadapter": {
        "changed": false,
        "failed": false,
        "network_data": {
            "0": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 1",
                "mac_addr": "00:50:56:82:c7:fb",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 7,
                "wake_onlan": true
            },
            "1": {
                "allow_guest_ctl": true,
                "connected": false,
                "device_type": "VMXNET3",
                "label": "Network adapter 3",
                "mac_addr": "00:50:56:58:59:61",
                "name": "VM Network",
                "start_connected": true,
                "unit_number": 9,
                "wake_onlan": true
            }
        }
    }
}

TASK [vmware_guest_network : assert the network adapter was disconnected] ************************************************************************************
task path: /home/goneri/.ansible/test/tmp/vmware_guest_network-w6orkuut-ÅÑŚÌβŁÈ/test/integration/targets/vmware_guest_network/tasks/main.yml:146
fatal: [testhost]: FAILED! => {
    "assertion": "disc_netadapter is changed",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed"
}

PLAY RECAP ***************************************************************************************************************************************************
testhost                   : ok=49   changed=17   unreachable=0    failed=1    skipped=8    rescued=0    ignored=1   

Command exited with status 2 after 90.65792679786682 seconds.
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/ ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.030608177185058594 seconds.
Run command: /usr/bin/python2 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python2
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.012304067611694336 seconds.
Run command: /usr/bin/python2.7 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python2.7
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.010623931884765625 seconds.
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.022822856903076172 seconds.
Run command: /usr/bin/python3.5 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python3.5
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.014861345291137695 seconds.
Run command: /usr/bin/python3.6 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python3.6
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.014163732528686523 seconds.
Run command: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3.7 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_da ...
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin/python3.7
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.022728681564331055 seconds.
Run command: /usr/bin/python3.8 /home/goneri/git_repos/ansible_projects/a/test/lib/ansible_test/_data/versions.py
Working directory: /home/goneri/git_repos/ansible_projects/a
Program found: /usr/bin/python3.8
HOME=/home/goneri
LC_ALL=en_US.UTF-8
PATH=/home/goneri/.fzf/bin:/home/goneri/git_repos/ansible_projects/a/bin:/home/goneri/git_repos/ansible_projects/a/.virtualenv/py37/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin:/usr/lib/golang/bin:/home/goneri/go/bin:/home/goneri/.cargo/bin:/home/goneri/.local/bin
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
Command exited with status 0 after 0.015594005584716797 seconds.
NOTICE: To resume at this test target, use the option: --start-at vmware_guest_network
ERROR: Command "ansible-playbook vmware_guest_network-25645cr0.yml -i inventory -vvvv" returned exit status 2.

Compilation exited abnormally with code 1 at Mon Sep 30 13:51:06
```

Reply:
Hi @goneri ,
In precheckin tests, all vmware_guest_network tests are skipped with below message
```
skipping: [testhost] => {"changed": false, "skip_reason": "Conditional result was False"}
```

I have added **connected** bool param for network adapter addition tests. Please can you rerun your integration suite and let me know?

Reply:
I still face the same problem :-( : https://paste.fedoraproject.org/paste/fPcbgoQwEjQ3O885S~2b5w

Reply:
Hi @goneri , Weird. Please can you share steps to run the tests as you do? My test runs are getting each tests skipped. Even in Pull request precheckin tests are skipping vmware_guest_network tests.
I can test before making changes.

Thanks

Reply:
I prepared my env as explain here: https://docs.ansible.com/ansible/devel/dev_guide/platforms/vmware_guidelines.html#testing-with-your-own-infrastructure

And run `ansible-test` this way:
`VMWARE_TEST_PLATFORM=static ansible-test integration --python 3.7 -vvvv vmware_guest_network`